docs/source/api-ref/mo.rst

```diff
@@ -13,11 +13,13 @@ profiles, and policies are logical entities represented as MOs.
 
 Accessing Properties
 ---------------------
-When you create a managed object (MO), you can access properties as follows::
+When you create a managed object (MO), you can access properties as follows:
```

Comment:
We need either a double colon or a '..code-block:: python' so syntax highlighting works in the doc build.